backend/src/main/java/com/bakdata/conquery/io/result/arrow/ArrowRenderer.java

```diff
@@ -50,7 +52,7 @@ public static void renderToStream(
             int batchsize) throws IOException {
         // Test the execution if the result is renderable into one table
         Stream<EntityResult> results = getResults(exec);
-        List<ResultInfo> resultInfos = getResultInfos(exec);
+        List<ResultInfo> resultInfos = getResultInfos(exec, cfg);
```

Comment:
Kannst du die I18n nicht mit dem Aufruf `generateFieldsFromResultType` in Zeile  59 erreichen?

Reply:
Sonst haben wir immer zwei Punkte an denen es in Betracht gezogen wird

Reply:
weiß nicht genau was du meinst

Reply:
Ah, die cfg wurde gar nicht mehr gebraucht